tests/wandb_integration_test.py

```diff
@@ -400,4 +400,4 @@ def mock_min_size(self, size):
 
     server_ctx = live_mock_server.get_ctx()
     print(server_ctx["file_bytes"], sent)
-    assert abs(server_ctx["file_bytes"] - sent) < 15000
+    assert abs(server_ctx["file_bytes"]["saveFile"] - sent) == 0
```

Comment:
having an assert for "saveFile" being in server_ctx will produce much cleaner test output in the case of an error... otherwise you get an exception instead of an assert failure.